### ISSUE TITLE
publishing to newly created channel results in 'is_mine' error

### ISSUE THREAD

**Issue by billbitt** [lbryio/spee.ch#332]
steps to recreate: 
 - go to  the spee.ch home page 
 - choose an asset to publish
 - select 'in a channel' and create a channel from the drop down (as opposed to logging into an existing one)
 - click publish

result should be the following error: 

``` {"error":{"code":-32500,"data":[" File \"site-packages/twisted/internet/defer.py\", line 1299, in _inlineCallbacks"," "," File \"site-packages/lbrynet/core/Wallet.py\", line 803, in _handle_claim_result"," ","exceptions.AssertionError: result in unexpected format:{'is_mine': False}"],"message":"result in unexpected format:{'is_mine': False}"},"id":null,"jsonrpc":"2.0"} ```

This may not be news to Jack, as  I believe he already figured out it was a race condition having to do with the channel creation, but I think he saw it only happening when a lot of channels are created at once.  It appears to happen for me each time a channel is newly created and published too.

**Comment by tzarebczan** [comment]
@shyba I don't think this has been an issue on speech since we updated it to use the id vs the channel as you mentioned. Funny thing is we did that for wallet performance reasons and not to fix this issue haha.
